bip-0360.mediawiki

```diff
@@ -315,8 +313,9 @@ keys from the transaction while still proving they were part of the original com
 This merkle tree construction creates an efficient cryptographic commitment to multiple public keys while enabling
 selective disclosure.
 
-This allows for inclusion of a Taproot MAST merkle root in the attestation, which makes P2QRH a quantum-resistant
-version of Taproot.
+This allows for inclusion of a [https://github.com/bitcoin/bips/blob/master/bip-0114.mediawiki BIP-114] Taproot
+Merkelized Abstract Syntax Tree (MAST) merkle root in the attestation, which makes P2QRH a quantum-resistant
+version of Taproot transactions.
```

Comment:
Taproot does not use an Abstract Syntax Tree. It uses an Alternative Script Tree. Also the term "MAST" never appears in the Taproot BIPs, they refer to the concept as "script tree".

Reply:
Updated here:
https://github.com/bitcoin/bips/pull/1670/commits/8c0f7982932e180a32539120445a629077072b40#diff-e296d32a8f60ef9f846683e801718d16c41fa7d8f1065e0afa6405a0764a91e3R315